app/ota_client.py

```diff
@@ -246,44 +248,51 @@ def __init__(self, info: str):
 
 
 class DirectoryInf(_BaseInf):
-    """
-    Directory file information class
-    """
+    """Directory file information class."""
+
+    __slots__ = ["path"]
 
     def __init__(self, info):
         super().__init__(info)
         self.path = Path(self.de_escape(self._left[1:-1]))
 
+        del self._left
+
 
 class SymbolicLinkInf(_BaseInf):
-    """
-    Symbolik link information class
-    """
+    """Symbolik link information class."""
 
     _pattern = re.compile(r"'(?P<link>.+)((?<!\')',')(?P<target>.+)'")
 
+    __slots__ = ["slink", "srcpath"]
+
     def __init__(self, info):
         super().__init__(info)
         res = self._pattern.match(self._left)
         assert res is not None
+        del self._left
```

Comment:
add blank line before `del self._left` same as DirectoryInf and RegularInf. Or remove all blank lines before it for each class.